src/ui/components/shared/UserSettingsModal/PreferencesSettings.tsx

```diff
@@ -63,11 +63,22 @@ function NotificationPreferences({
 }
 
 function PrivacyPreferences() {
-  const toggleDisableLogRocket = () => {
-    prefs.disableLogRocket = !prefs.disableLogRocket;
-    setDisableLogRocket(prefs.disableLogRocket);
+  const [disableLogRocket, setDisableLogRocket] = useState<boolean>();
+  const { userSettings, loading } = hooks.useGetUserSettings();
+  const updateLogRocket = hooks.useUpdateUserSetting("disableLogRocket", "Boolean");
+
+  const toggle = (newValue: boolean) => {
+    setDisableLogRocket(newValue);
+    updateLogRocket({
+      variables: {
+        newValue,
+      },
+    });
   };
-  const [disableLogRocket, setDisableLogRocket] = useState(prefs.disableLogRocket);
+
+  useEffect(() => {
+    setDisableLogRocket(userSettings.disableLogRocket);
```

Comment:
Looks like it's possible for `useGetUserSettings` to return `loading: true` here since it's not always being called in the parent. It is if you open the debugger, but if you stay in the viewer you'll hit loading here.

This currently allows this UI to default to give the impression that LogRocket is disabled while loading. What do you think about not showing `PrivacyPreferences` until after loading, so at least we're not giving the wrong impression?